tmt/steps/execute/upgrade.py

```diff
@@ -276,8 +276,8 @@ def _perform_upgrade(self, guest: tmt.steps.provision.Guest) -> None:
                 # Instead, we want it to re-use existing, already cloned path.
                 # ignore[typeddict-item]: data is _RwStepData, we do not have more detailed type
                 # for raw step data of internal/upgrade plugins, it would be pretty verbose.
-                data['url'] = None  # type: ignore[typeddict-item]
-                data['path'] = self._discover_upgrade.testdir  # type: ignore[typeddict-item]
+                data['url'] = None   # type: ignore[typeddict-unknown-key]
+                data['path'] = self._discover_upgrade.testdir  # type:ignore[typeddict-unknown-key]
```

Comment:
Please, update also the explanatory comment above, to match the changed error code.